pkgs/top-level/all-packages.nix

```diff
@@ -12170,6 +12170,7 @@ in
   jemalloc = callPackage ../development/libraries/jemalloc { };
 
   jemalloc450 = callPackage ../development/libraries/jemalloc/jemalloc450.nix { };
+  jemalloc450-mysql = callPackage ../development/libraries/jemalloc/jemalloc450.nix { disableInitExecTls = true; };
```

Comment:
```suggestion
  jemalloc-mysql = callPackage ../development/libraries/jemalloc/jemalloc450.nix { disableInitExecTls = true; };
```

Reply:
If this jmalloc is only used in mariadb, we should actually rename it to `-mariadb`.